leelastar/network/layers.py

```diff
@@ -0,0 +1,76 @@
+"""
+@file: layers.py
+Created on 02.02.19
+@project: LeelaStar
+@author: kiudee
```

Comment:
I think the meta info adds unnecessary clutter as it can be simply looked up via git history

Reply:
I think it adds practicality, I think its easier to look at this info directly on the file. Even tho you can look at it on git

Reply:
What about having an `AUTHORS` file in the directory root?
It could be constructed using:
```git shortlog -sn | cut -c8-```

Reply:
yeah, I think that's best, so keep the module description and remove/move away the rest